docs/plans/2025-11-29-badge-standardization.md

```
@@ -0,0 +1,1316 @@
+| QUEUED | NEUTRAL | Queued | bi-hourglass-split |
+| DOWNLOADING | PROCESSING | Downloading | (spinner) |
+| COMPLETED | SUCCESS | Downloaded | bi-check-circle |
+| FAILED | DANGER | Download Failed | bi-exclamation-triangle |
+
+### Hash Verification
+
+| State | BadgeType | Text | Icon |
+|-------|-----------|------|------|
+| Verified | SUCCESS | Hash Verified | bi-shield-check |
+| Mismatch | DANGER | Hash Mismatch | bi-shield-x |
+| Unverified | WARNING | Hash Unverified | bi-shield-exclamation |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)